### PR TITLE
Weights metad

### PR DESCRIPTION
**Pull request by dilnoza92**
plumed_driver method was added to peptidesim that takes a metadynamics folder path and metadynamics plumed file and generates WEIGHTS file in that folder. #58

### CODE REVIEW

**Comment by whitead** [comment]
Does this supersede #64 or is this separate?

**Comment by whitead** [comment]
Does this supersede #64 or is this separate @dilnoza92 ?

**Comment by dilnoza92** [comment]
@whitead they are separate.

File: package/peptidesim/peptidesim.py [hunk 1]
```diff
@@ -540,6 +544,142 @@ class Demux(gromacs.core.Command):
...
+
+                if len(line_segments) == 0 or len(line_segments) == 1:
+                    next
+                elif line_segments[0].startswith('METAD') and len(line_segments) < 3:
+                    meta_start_number = number
+                    metad_lines.append([number, line_segments, line])
+
+                elif line_segments[1] == 'METAD' and line_segments[0] == '...':
+                    metad_lines.append([number, line_segments, line])
+                    exit
```
Comment (whitead) [inline package/peptidesim/peptidesim.py]: Probably don't actually want to exit the program

File: package/peptidesim/peptidesim.py [hunk 2]
```diff
@@ -540,6 +544,142 @@ class Demux(gromacs.core.Command):
...
+        with open(plumed_metad, 'r') as f:
+            lines = f.readlines()
+            CVS = []
+
+            for line in lines:
+
+                line_segments = line.strip().split()
+
+                if len(line_segments) == 0 or len(line_segments) == 1:
+                    next
```
Comment (whitead) [inline package/peptidesim/peptidesim.py]: this is not a valid python statement

**Comment by whitead** [comment]
@dilnoza92 I took a close look and I cannot get the test to pass. It seems to be missing the lines that define the CVs. Can you take a look at the method to see if you can figure it out?

**Pull request closed without merging**